src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -524,4 +534,96 @@ protected function getObject()
 
         return $obj;
     }
+
+    /**
+     * @test
+     */
+    public function testEncodeXmlWithBoolValue()
+    {
+        $expectedXml = '<?xml version="1.0"?>
```

Comment:
You may use the heredoc syntax (the IDE will be able to highlight the snippet). (Just suggestion, not mandatory).